core/src/main/res/values/strings.xml

```diff
@@ -106,11 +106,11 @@
     <string name="noprofileset">NO PROFILE SET</string>
     <string name="date">Date</string>
     <string name="units_label">Units</string>
-    <string name="dia_label">DIA</string>
-    <string name="ic_label">IC</string>
-    <string name="isf_label">ISF</string>
-    <string name="basal_label">Basal</string>
-    <string name="target_label">Target</string>
+    <string name="dia_label">Duration of Insulin Action</string>
+    <string name="ic_label">Insulin to Carb ratio</string>
+    <string name="isf_label">Insulin Sensitivity Factor</string>
+    <string name="basal_label">Basel rate</string>
+    <string name="target_label">Blood Glucose Target</string>
```

Comment:
at least `target_label` is used elsewhere. Making the string long breaks something